src/components/gitProvider/GitProvider.tsx

```diff
@@ -284,11 +288,11 @@ function CollapsedList({
                             placement="bottom"
                             content={enabled ? 'Disable git account' : 'Enable git account'}
                         >
-                            <span style={{ marginLeft: 'auto' }} data-testid={`${name}-toggle-button`}>
+                            <span onClick={stopPropagation} style={{ marginLeft: 'auto' }} data-testid={`${name}-toggle-button`}>
                                 {loading ? (
                                     <Progressing />
                                 ) : (
-                                    <List.Toggle onSelect={(en) => toggleEnabled(en)} enabled={enabled} />
+                                    <List.Toggle onSelect={(en) => setToggleEnabled(en)} enabled={enabled} />
```

Comment:
can we directly write onSelect={toggleEnabled}?

Reply:
done